Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -264,8 +264,9 @@ private func sandboxProfile() -> String {
     // These are required by the Swift compiler.
     stream <<< "(allow process*)" <<< "\n"
     stream <<< "(allow sysctl*)" <<< "\n"
-    stream <<< "(allow mach*)" <<< "\n"
-    stream <<< "(allow signal)" <<< "\n"
+    stream <<< "(allow mach-lookup"
+    stream <<< "    (global-name \"com.apple.system.opendirectoryd.membership\")"
+    stream <<< ")"
```

Comment:
Can we put a comment here about why we even need this? Do you know what requires it (I am going off of nothing except for the name)?